tpm2/constants.go

```diff
@@ -223,28 +223,30 @@ const (
 	cmdCertify            tpmutil.Command = 0x00000148
 	cmdCertifyCreation    tpmutil.Command = 0x0000014A
 	cmdReadNV             tpmutil.Command = 0x0000014E
-	cmdCreate             tpmutil.Command = 0x00000153
-	cmdLoad               tpmutil.Command = 0x00000157
-	cmdQuote              tpmutil.Command = 0x00000158
-	cmdSign               tpmutil.Command = 0x0000015D
-	cmdUnseal             tpmutil.Command = 0x0000015E
-	cmdContextLoad        tpmutil.Command = 0x00000161
-	cmdContextSave        tpmutil.Command = 0x00000162
-	cmdFlushContext       tpmutil.Command = 0x00000165
-	cmdLoadExternal       tpmutil.Command = 0x00000167
-	cmdMakeCredential     tpmutil.Command = 0x00000168
-	cmdReadPublicNV       tpmutil.Command = 0x00000169
-	cmdReadPublic         tpmutil.Command = 0x00000173
-	cmdStartAuthSession   tpmutil.Command = 0x00000176
-	cmdGetCapability      tpmutil.Command = 0x0000017A
-	cmdGetRandom          tpmutil.Command = 0x0000017B
-	cmdHash               tpmutil.Command = 0x0000017D
-	cmdPCRRead            tpmutil.Command = 0x0000017E
-	cmdPolicyPCR          tpmutil.Command = 0x0000017F
-	cmdReadClock          tpmutil.Command = 0x00000181
-	cmdPCRExtend          tpmutil.Command = 0x00000182
-	cmdPolicyGetDigest    tpmutil.Command = 0x00000189
-	cmdPolicyPassword     tpmutil.Command = 0x0000018C
+	// CmdPolicySecret must be public to compute the default AuthPolicy value.
```

Comment:
This appears in public godoc. Let's change the comment to:
`CmdPolicySecret is a command code for TPM2_PolicySecret. It's exported for computing of default AuthPolicy value.`

Reply:
Done